lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -136,33 +136,51 @@ sub get_first_collection {
     return $self->sql->select_single_value($query, $editor_id);
 }
 
-sub find_all_by_editor {
```

Comment:
I didn't know that it did. Don't have an immediate answer to that, needs investigating. :)